### PR TITLE
duden: 0.18.0 -> 0.19.1

### PR DESCRIPTION
**Pull request by AndersonTorres**
## Description of changes

Closes https://github.com/NixOS/nixpkgs/pull/292064 and https://github.com/NixOS/nixpkgs/pull/291670

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by AndersonTorres** [comment]
@linuxissuper are you interested in co-maintainership?

**Comment by linuxissuper** [comment]
> @linuxissuper are you interested in co-maintainership?

yes, thanks for asking  
should I create a new MR for this or do you just add my name?

**Comment by afh** [comment]
Result of `nixpkgs-review pr 292193` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>duden (python311Packages.duden)</li>
    <li>duden.dist (python311Packages.duden.dist)</li>
  </ul>
</details>

File: pkgs/development/python-modules/duden/default.nix [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      requests
+      setuptools
+    ];
+
+    pythonImportsCheck = [ "duden" ];
+
+    meta = {
+      homepage = "https://github.com/radomirbosak/duden";
+      changelog = "https://github.com/radomirbosak/duden/blob/${finalAttrs.src.rev}/CHANGELOG.md";
+      description = "CLI for http://duden.de dictionary written in Python";
```
Comment (afh) [inline pkgs/development/python-modules/duden/default.nix:37]:
```suggestion
      description = "CLI for https://duden.de dictionary written in Python";
```

**Review by afh — APPROVED** [review]
Nice work, @AndersonTorres, well done!

Minor suggestion to use https in `meta.description`, see comment below.

**Comment by AndersonTorres** [comment]
> > @linuxissuper are you interested in co-maintainership?
> 
> yes, thanks for asking should I create a new MR for this or do you just add my name?

I will just add your tag to maintainers list.

**Comment by linuxissuper** [comment]
Result of `nixpkgs-review pr 292193` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>duden (python311Packages.duden)</li>
    <li>duden.dist (python311Packages.duden.dist)</li>
  </ul>
</details>

**Review by linuxissuper — APPROVED** [review]
LGTM

but a question (maybe stupid one 😅): why get rid of `rec`?

**Comment by AndersonTorres** [comment]
https://nix.dev/guides/best-practices.html#recursive-attribute-set-rec

**Review by linuxissuper — APPROVED** [review]

**Review by leona-ya — APPROVED** [review]

**Comment by afh** [comment]
Thanks for merging, @leona-ya, much appreciated 🙂

**Pull request merged**